src/Sarif.TestUtilities/FileDiffingTests.cs

```diff
@@ -198,7 +198,7 @@ protected static string GenerateDiffCommand(string suiteName, string expected, s
             return fullPath;
         }
 
-        protected static bool AreEquivalentSarifLogs<T>(string actualSarif, string expectedSarif, IContractResolver contractResolver = null)
+        public static bool AreEquivalentSarifLogs<T>(string actualSarif, string expectedSarif, IContractResolver contractResolver = null)
```

Comment:
>AreEquivalentSarifLogs [](start = 27, length = 22)

this helper likely should belong somewhere else, not this base class. but maybe it should stay here, if we end up converting everything to a filediffing-derived test class. we'll keep an eye on it.

Reply:
string extension. `actualSarif.IsSarifEquivalentOf(expectedSarif)`.

---
In reply to: [247995751](https://github.com/Microsoft/sarif-sdk/pull/1213#discussion_r247995751) [](ancestors = 247995751)